.github/workflows/build-test.yml

```diff
@@ -48,23 +53,27 @@ jobs:
         run: |-
           git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
           git checkout scratch
-      - name: Set up JDK 11
-        uses: olafurpg/setup-scala@v10
-        with:
-          java-version: adopt@1.11.0-9
+
       - name: Cache Coursier cache
-        uses: coursier/cache-action@v5
+        uses: coursier/cache-action@v6.4.0
+
+      - name: Set up JDK 17
```

Comment:
this is setting up 11 not 17

Reply:
might be better to have this with 8 so that we validate that it's compiling with 8, and then tests can be run with 11 (and 17)